app/src/main/java/app/grapheneos/camera/ui/activities/CaptureActivity.kt

```diff
@@ -40,10 +43,12 @@ open class CaptureActivity : MainActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        retakeIcon = findViewById(R.id.retake_icon)
-        flipCameraContent = findViewById(R.id.flip_camera_icon_content)
+        binding = ActivityMainBinding.inflate(layoutInflater)
```

Comment:
`CaptureActivity` is a subclass of the `MainActivity`, `ActivityMainBinding.inflate` is called during `super.onCreate(savedInstanceState)`